centreon/src/Core/HostTemplate/Application/UseCase/PartialUpdateHostTemplate/PartialUpdateHostTemplate.php

```diff
@@ -165,6 +179,199 @@ private function updatePropertiesInTransaction(PartialUpdateHostTemplateRequest
         }
     }
 
+    /**
+     * @param PartialUpdateHostTemplateRequest $request
+     * @param HostTemplate $hostTemplate
+     *
+     * @throws \Throwable|AssertionFailedException|HostTemplateException
+     */
+    private function updateHostTemplate(PartialUpdateHostTemplateRequest $request, HostTemplate $hostTemplate): void
+    {
+        $this->info('PartialUpdateHostTemplate: update host template', ['host_template_id' => $hostTemplate->getId()]);
+
+        $inheritanceMode = $this->optionService->findSelectedOptions(['inheritance_mode']);
+        $inheritanceMode = isset($inheritanceMode['inheritance_mode'])
+            ? (int) $inheritanceMode['inheritance_mode']->getValue()
+            : 0;
+
+        if (
+            ! $request->name instanceOf NoValue
+            && $hostTemplate->getName() !== HostTemplate::formatName($request->name)
```

Comment:
Move this in assertIsValidName
```suggestion
            && $hostTemplate->getName() !== HostTemplate::formatName($request->name)
```